apiserver/facades/agent/uniter/uniter_test.go

```diff
@@ -4095,3 +4095,17 @@ func (s *uniterSuite) TestNetworkInfoCAASModel(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 	c.Check(result.Results["db"], jc.DeepEquals, expectedResult)
 }
+
+// TODO (agprado): Next PR diferenciate between Unit and App
```

Comment:
If you follow the implementation example in AssignedMachine() you can literally cut and paste the TestAssignedMachine code here and tweak it.